src/howto_private_tx.md

```diff
@@ -32,7 +32,40 @@ cd substraTEE-worker
 git checkout v0.6.12-sub2.0.0
 make
 # this might take 10min+ on a fast machine
+```
+
+In case you get the error:
+```bash
+error: failed to get `sgx-runtime` as a dependency of package `substratee-stf v0.6.12-sub2.0.0 (/root/work/substraTEE-worker/stf)`
+Caused by:
+  failed to load source for dependency `sgx-runtime`
+```
+you need to edit the file Cargo.lock manually. Within the file, find the inclusion of the package sgx-runtime. It should look similiar to:
```

Comment:
I also had to edit `enclave/Cargo.lock`

Reply:
Then as a suggestion: because this does not seem to be a rare error, lets move the error handling to the troubleshooting section in a more generalised form.
For the listed errors (sgx-runtime in Cargo.lock and enclave/Cargo.lock) within the substraTEE-worker I will make a PR.